pkg/virt-handler/isolation/process_test.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package isolation
+
+import (
+	"github.com/mitchellh/go-ps"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var emptyProcessList = []ps.Process{}
+
+var _ = Describe("process", func() {
+	const proc1Andproc5Exec = "processA"
+	const proc2Andproc4Pid = 110
+	const nonExistPPid = 300
```

Comment:
nit: We usually use a const block.
```
const (
...
)
```

I would also not embed two things into one name.
If we take `proc1Andproc5Exec`, then you can just call it: `processTestExecPath`.
Some for the pid one, processTestPID.

Reply:
Done